addons.yaml

```diff
@@ -181,3 +181,11 @@ microk8s-addons:
       confinement: "classic"
       supported_architectures:
         - amd64
+
+    - name: "easyhaproxy"
+      description: "EasyHAProxy can configure HAProxy automatically based on ingress labels"
+      version: "latest"
```

Comment:
Could we deploy a specific version?

We would like to provide a consistent UX experience. It would be a bad experience if the user got a different version with no good justification from the same MicroK8s version.

Also as time goes by you may want to do a backwards incompatible change resulting in the addon breaking on old MicroK8s releases. For example after some K8s releases, lets say on 1.32 you may want to push a change that and stop supporting all K8s releases prior to 1.28. If the addon on the 1.27 track uses the latest version it will break.

Reply:
After 1.27 comes out and before the 1.28 release we will create a "1.27" branch on the addons repository and all the 1.27 releases (1.27.1, 1.27.2 etc)  we will be using this branch.